src/Language/PureScript/Make.hs

```diff
@@ -138,10 +149,11 @@ make ma@MakeActions{..} ms = do
   (buildPlan, newCacheDb) <- BuildPlan.construct ma cacheDb (sorted, graph)
 
   let toBeRebuilt = filter (BuildPlan.needsRebuild buildPlan . getModuleName . CST.resPartial) sorted
+  let totalModuleCount = length toBeRebuilt
```

Comment:
I chose `toBeRebuilt` as the `x` in `[n of x] Compiling ...` as IMO, it provides a better overview of linear progress. I've also yet to figure out how to do GHC/Stack-style indices where `n` is the actual position of the module in the build list.